src/transport/UnauthenticatedSessionTable.h

```diff
@@ -113,7 +114,8 @@ class UnauthenticatedSession : public Session,
 
     bool IsPeerActive() const
     {
-        return ((System::SystemClock().GetMonotonicTimestamp() - GetLastPeerActivityTime()) < kMinActiveTime);
+        return ((System::SystemClock().GetMonotonicTimestamp() - GetLastPeerActivityTime()) <
+                GetRetransmissionTimeout(mRemoteMRPConfig.mActiveRetransTimeout));
```

Comment:
Hmm. Was it already updated in the spec? I think it still refers to SLEEPY_ACTIVE_THRESHOLD constant.

Reply:
> SLEEPY_ACTIVE_THRESHOLD

This is actually my bad. Thanks for correcting me. I have not noticed that the `4000ms` value was defined as a constant. I thought `SLEEPY_ACTIVE_THRESHOLD` was not explicitly defined (and the GitHub UI was not showing me the `Parameters and Constants` results :/ )

I have updated the code to add back `kMinActiveTime` and I have added an other parameter to `GetRetransmissionTimeout`.